### DISCUSSION TITLE
The directory Files could not be created.

### DISCUSSION THREAD

**Discussion by rabhab20** [TruCopilot/phpfastcache#805] [General]
Hello. I get the error:
"The directory /tmp/phpfastcache/insta.kaspsoft.com/Files could not be created.
Additional information provided by error_get_last():
mkdir(): File exists in /home/i/imranh4k/insta.kaspsoft.com/public_html/vendor/phpfastcache/phpfastcache/lib/Phpfastcache/Core/Pool/IO/IOHelperTrait.php line 170"
Using raiym/instagram-php-scraper

**Comment by Geolim4** [comment]
Please use issues for reporting any bugs.
